rero_ils/modules/documents/views.py

```diff
@@ -574,3 +593,50 @@ def in_collection(item_pid):
         .filter('term', published=True)
         .scan()
     )
+
+
+@blueprint.app_template_filter()
+def document_types(record, translate=True):
+    """Get docuement types.
+
+    :param record: record
+    :param translate: translate document type
+    :return: dictonary list of document types
+    """
+    doc_types = record.document_types
+    if translate:
+        for idx, doc_type in enumerate(doc_types):
+            doc_types[idx] = _(doc_type)
+    return doc_types
```

Comment:
```python
if translate:
  return [_(doc_type) for doc_type in record.document_types]
else:
  return record.document_types 
```
If only Python has an easy-readable ternary instruction :'(

Reply:
`return [_(doc_type) for doc_type in record.document_types] if translate else record.document_types`